client/src/components/common/Input/index.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { memo } from "react";
+import mailLogo from "../../../assets/icon/mail.png";
+import './style.css'
+
+interface InputPropsCus extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'size'> {
+  label?: string;
+  className?: string;
+  size?: "large" | "small" | "medium" | string;
+  icon?: string;
+  placeholder?: string;
+  type: string
+}
+
+
+export const Input: React.FC<InputPropsCus> = memo(
+  ({ label, className, size, placeholder,type, ...rest }) => {
```

Comment:
Check code format

Reply:
what did you mean? can you tell me more detail?

Reply:
Just saw placeholder and type is have not spacing